articles/quickstart/spa/_includes/_install_auth0js.md

```diff
@@ -1,7 +1,7 @@
 ## Install auth0.js
 
-Integrating Auth0 in your application requires the **auth0.js** library. Install it using npm or yarn.
-
+You need the auth0.js library to integrate Auth0 into your application.
+Install auth0.js.
```

Comment:
All the code in the articles has an introductory sentence in this format: "Do x.", so I felt like this doesn't need more introduction (just to keep consistency).